platforms.yaml

```
@@ -0,0 +1,63 @@
+# This file specifies image customizations that are specific to particular
+# architecture/platform pairs.  It is applied (indirectly, via
+# /boot/coreos/platforms.json) by gf-set-platform at build time and
+# coreos-installer at install time (if --platform is specified).  qemu and
+# metal don't use gf-set-platform and ignore this file.
+#
+# Currently this is used to configure the default console.  For any
+# arch/platform pairs not specified, GRUB and the kernel will apply their
+# own defaults.
+#
+# All architectures, platforms, and fields are optional.
+aarch64:
+  aws:
+    # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/connect-to-serial-console.html#sc-considerations
+    grub_commands:
+      - serial --speed=115200
+      - terminal_input serial
+      - terminal_output serial
+    kernel_arguments:
+      - console=ttyS0,115200n8
+  packet:
+    # https://metal.equinix.com/developers/docs/resilience-recovery/serial-over-ssh/#limitations
+    grub_commands:
+      - serial --speed=115200
+      - terminal_input serial
+      - terminal_output serial
+    kernel_arguments:
+      - console=ttyAMA0,115200
+x86_64:
+  legacy: true  # continue using legacy behavior for now
```

I'd prefer the below to just be commented out to make it more obvious, but a substitute would be adding a few more words here

```suggestion
  legacy: true  # continue using legacy behavior for now, below config will not be used
```

---

As is, only one line needs to be changed to test the new behavior.  I've added an additional comment though.